packages/web/src/components/Header/index.tsx

```diff
@@ -0,0 +1,54 @@
+import classNames from "classnames";
+import { DefaultButton } from "@fluentui/react";
+import * as React from "react";
+import { Link, useLocation } from "react-router-dom";
+import styles from "./Header.module.css";
+import HelpMenu from "./Help";
+import { APPLICATION_NAME } from "../../constants";
+import AICSLogo from "../../../assets/AICS-logo-and-name.svg";
```

Comment:
Imports in our JS repos are ordered like so:
```
import { DefaultButton } from "@fluentui/react";
import classNames from "classnames";
import * as React from "react";
import { Link, useLocation } from "react-router-dom";

import HelpMenu from "./Help";
import { APPLICATION_NAME } from "../../constants";
import AICSLogo from "../../../assets/AICS-logo-and-name.svg";

import styles from "./Header.module.css";
```
AKA
```
<alphabetical third party imports>

<local imports ordered by how far away they are from this package (like `../../` vs `../../../../..`)>

<style sheet if applicable>
```

Reply:
Fixed in [cadb6d6](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/107/commits/cadb6d6ce41eb74806b71dbfc7747b5b3f533c33)